README.md

```diff
@@ -1,34 +1,89 @@
-# kafka-connect-datagen
+# Overview
 
-`kafka-connect-datagen` is a Kafka Connect connector for generating mock data, not suitable for production
+`kafka-connect-datagen` is a Kafka Connect connector for generating mock data.
+It is available in [Confluent Hub](https://www.confluent.io/connector/kafka-connect-datagen/).
+It is not suitable for production.
 
-## Local
+# Confluent Platform running on local install
+
+## Install connector from Confluent Hub
+
+You may install the `kafka-connect-datagen` connector from [Confluent Hub](https://www.confluent.io/connector/kafka-connect-datagen/).
+
+```bash
+confluent-hub install --no-prompt confluentinc/kafka-connect-datagen:0.1.0
+```
+
+## Build connector from latest code
+
+Alternatively, you may build and install the `kafka-connect-datagen` connector from latest code.
 
 ```bash
 git checkout v0.1.0
-confluent destroy
 mvn clean compile package
 confluent-hub install --no-prompt target/components/packages/confluentinc-kafka-connect-datagen-0.1.0.zip
+```
+
+## Run connector in local install
+
+Here is an example of how to run the `kafka-connect-datagen` on a local install:
+
+```bash
+confluent destroy
 confluent start connect
 sleep 15
```

Comment:
Is this necessary? I thought the CLI automatically waited for the Connect framework to come up when the `start connect` command is invoked.

Reply:
Still needs a wait.